packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -303,6 +303,18 @@ export interface JobDefinitionProps {
    * @default - EC2
    */
   readonly platformCapabilities?: PlatformCapabilities[];
+
+  /**
+   * Specifies whether to propagate the tags from the job or job definition to the corresponding Amazon ECS task.
+   * If no value is specified, the tags aren't propagated.
+   * Tags can only be propagated to the tasks during task creation. For tags with the same name,
+   * job tags are given priority over job definitions tags.
+   * If the total number of combined tags from the job and job definition is over 50, the job is moved to the `FAILED` state.
```

Comment:
Is this validation we can do at synth time instead of letting the job fail at deploy time? If so, it should be added.

Reply:
I'm not aware of a way to get the tag count of a construct; the current Tag aspect API only offers the ability to add/remove tags. If you have any ideas happy to dig deeper.

Reply:
Bummer, that's OK.